packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3924,6 +3937,11 @@ class InputDecorationTheme with Diagnosticable {
   ///  * [helperMaxLines], the equivalent but for the [InputDecoration.helperText].
   final int? errorMaxLines;
 
+  /// {@macro flutter.material.inputDecoration.floatingLabelGap}
```

Comment:
This macro doesn't exist, to define it you will have to use `{@template flutter.material.inputDecoration.floatingLabelGap} ... {@endtemplate}` somewhere else in a doc (see other examples of `{@template` in our repo)